### DISCUSSION TITLE
Ansible issue while installing TPOT.

### DISCUSSION THREAD

**Discussion by Sagarshar06** [telekom-security/tpotce#1532] [Q&A]
Getting following error while installing service. What is the work around and root cause of this error?

![error](https://github.com/telekom-security/tpotce/assets/152416657/353b2660-6fe5-4441-abf5-b6e26998c7cf)

**Comment by FloppyDucks** [comment]
Are you still seeing this issue? if so what do the ~/install_tpot.log files show when you read them? does anything stick out to you? 
Also what did you cross out from the picture after The error appears to have been in '/ [UNREADABLE]': line 720, column 7? what did you cross out? is it sensitive? because that line might be a clue. let me know if this helps or if this just confused you

**Comment by asvin00** [comment]
Same goes with me !! Did anyone find the solution ?


ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '/home/student/tpotce/installer/install/tpot.yml': line 174, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


    - name: Install grc from remote repo (AlmaLinux, Rocky)
      ^ here

**Reply by FloppyDucks** [reply]
are you willing to upload a copy of your tpot.yml to this discussion so I can see if anything is changed or misspelled compared to version I am running

**Reply by Stazeymitch** [reply]
hello i need help with the same issue

**Reply by Lafead** [reply]
You might need to re install, could be a misconfiguration. What server are you setting up on?

**Comment by YKTV8** [comment]
Ran across the similar error with a different line and column. I recently installed on a AWS instance running Debian 10 Buster. Uninstalled after the first failed install ( used ./install.sh --type=user) then on the 2nd install attempt same syntax error message ( this time ran ./install.sh).

Checked the file via nano on line 174 and column 7.

> ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.
> 
> The error appears to have been in '<REDACTED_PATH>: line 174, column 7, but may
> be elsewhere in the file depending on the exact syntax problem.
> 
> The offending line appears to be:
> 
> 
>     - name: Install grc from remote repo (AlmaLinux, Rocky)
>       ^ here
>
